### ISSUE TITLE
Unit tests of Configuratoin/Geometry failing in CMSSW_12_0_X_2021-07-19-1100

### ISSUE THREAD

**Issue by iarspider** [cms-sw/cmssw#34552]
Log file: https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc900/CMSSW_12_0_X_2021-07-19-1100/unitTestLogs/Configuration/Geometry#/

Failing tests: test2021Geometry, test2026Geometry

Error message:
```
Traceback (most recent call last):
  File "/data/cmsbld/jenkins/workspace/ib-run-qa/CMSSW_12_0_X_2021-07-19-1100/src/Configuration/Geometry/scripts/generate2021Geometry.py", line 1, in <module>
    from Configuration.Geometry.dict2021Geometry import *
ImportError: No module named Configuration.Geometry.dict2021Geometry
```
(similar message for test2026Geometry)

Most likely caused by dropping python2.

**Comment by qliphy** [comment]
assign geometry

**Comment by cvuosalo** [comment]
It looks like this issue is being fixed by #34554.

**Comment by cvuosalo** [comment]
+1

**Comment by cvuosalo** [comment]
Fixed by #34534
